_includes/components/banner.html

```diff
@@ -16,6 +16,11 @@
       </div>
     </header>
     <div class="usa-banner__content usa-accordion__content" id="gov-banner">
+      <script type="text/javascript">
+        // Hide the gov banner immediately rather than waiting for external JS
+        // sources to load and execute.
+        document.getElementById("gov-banner").setAttribute("hidden", "");
+      </script>
```

Comment:
would it make sense to just hardcode `hidden` in the HTML, and the JS could un-hide it later if needed?
```suggestion
    <div class="usa-banner__content usa-accordion__content" id="gov-banner" hidden>
```

Reply:
See Andrew's [earlier comment](https://github.com/18F/uswds-jekyll/pull/257#issuecomment-1448852449), but the gist is doing it with JS allows the banner to be expanded for users who have JS disabled.